registrations/views.py

```diff
@@ -363,6 +363,64 @@ def get(self, request, *args, **kwargs):
         return Response(resp, status=status)
 
 
+class JembiFacilityCheckHealthcheckView(APIView):
+
+    """ Jembi Facility Check Healthcheck Interaction
+        GET - returns service up - getting auth'd requires DB
+    """
+
+    permission_classes = (IsAuthenticated,)
+
+    def get(self, request, *args, **kwargs):
+        if not (
+            settings.JEMBI_BASE_URL
+            and settings.JEMBI_USERNAME
+            and settings.JEMBI_PASSWORD
+        ):
+
+            return Response(
+                "Jembi integration is not configured properly.",
+                status.HTTP_503_SERVICE_UNAVAILABLE,
+            )
+
+        try:
+            clinic_code = request.query_params.get("clinic_code")
+            result = requests.get(
+                urljoin(settings.JEMBI_BASE_URL, "NCfacilityCheck"),
+                headers={"Content-Type": "application/json"},
+                auth=(settings.JEMBI_USERNAME, settings.JEMBI_PASSWORD),
+                params={"criteria": "value:{}".format(clinic_code)},
+                verify=False,
+            )
+            result.raise_for_status()
+
+            jembi_result = result.json()
+
+            if len(jembi_result.get("rows")) > 0:
+                resp = {
+                    "up": True,
+                    "result": {"Facility": jembi_result.get("rows")[0][2]},
+                }
+            else:
+                raise Http404()
+
+        except (requests.exceptions.HTTPError,) as e:
+            if e.response.status_code == 400:
+                logger.warning(
+                    "400 Error when posting to Jembi.\n"
+                    "Response: %s\nPayload:%s" % (e.response.text)
+                )
+                return Response(
+                    "Error when posting to Jembi. Body: %s Payload: %r"
+                    % (e.response.content),
+                    status=status.HTTP_400_BAD_REQUEST,
+                )
+            else:
+                raise e
```

Comment:
We should probably handle the exception and return a response here. We don't want sentry alerts because their API is down, it's not this service that's broken.